pkg/backends/zenoss/zenoss.go

```diff
@@ -37,12 +41,15 @@ const (
 	paramDisableTLS          = "disable-tls"
 	paramInsecureTLS         = "insecure-tls"
 	paramAPIKey              = "api-key"
+	paramMaxRequests         = "max_requests"
```

Comment:
Shouldn't this be "max-requests" with a hyphen instead of underscore for consistency?